src/components/sidemenu/SideMenuPanel.js

```diff
@@ -42,11 +48,9 @@ export const deleteAccountDialog = ({ API, showDialog, store, theme }) => {
             await Promise.all([AsyncStorage.clear(), API.deleteWalletFromW3Site(token)]).catch(e =>
               log.error('Error deleting account', e.message, e)
             )
-            store.set('loadingIndicator')({ loading: false })
             window.location = '/'
           } else {
             showDialog('Error deleting account')
-            store.set('loadingIndicator')({ loading: false })
```

Comment:
i dont think this should be removed

Reply:
This code has been deleted since we don’t show the loader and we don’t hide it

Reply:
yeah but this is the case an error happens, so the loading state should be removed , so UI isnt kept in frozen state

Reply:
**showDialog**  Delete message and show error message

Reply:
but the loading indicator will still be  in loading state this blocks the whole UI app because the loadingindicator is a global flag

Reply:
Loading indicator has been removed

![image](https://user-images.githubusercontent.com/8187382/67942283-2d229d00-fbe0-11e9-8880-e0e97b56b221.png)